src/index.js

```diff
@@ -315,84 +310,76 @@ class ImageMinimizerPlugin {
       )
     ).flat();
 
-    const cpus = os.cpus() || { length: 1 };
-    const limit = this.options.concurrency || Math.max(1, cpus.length - 1);
-
+    // In some cases cpus() returns undefined
+    // https://github.com/nodejs/node/issues/19022
+    const limit = this.options.concurrency ?? os.cpus()?.length ?? 1;
```

Comment:
`0` is invalid, but now you can set it

Reply:
Yeah, sometimes it better to have more sticky checks, especial in popular solutions

Reply:
Done